net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/ItemController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+
+ *    THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+ *    INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+ *    AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *    AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+ *    OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ *    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ *    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ *    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ *    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *    POSSIBILITY OF SUCH DAMAGE.
+ */
+
+namespace OPNsense\UDPBroadcastRelay;
+
+/**
+ * Class ItemController
+ * @package OPNsense\UDPBroadcastRelay
+ */
+class ItemController extends \OPNsense\Base\IndexController
```

Comment:
Where is ItemController used? Didn't fint it. Maybe you can delete this file (after testing)

Reply:
It was there in the 'borrowed' source. I expect it's part of a template so I left it for future projects.

> I only had a quick view, some things can be deleted, or blindly copied. You put much effort in it, really cool! :) Can you also please add your copyright to every file? I know you just copied it, but that's ok. Every php and volt needs one. Binary pull is dirty, can you post your repo and maybe try to add port to opnsense ports?

I can put it there, with a link to this. @fichtner will need to do his magic on it though as it's a very basic makefile etc.